accelerator/config.h

```diff
@@ -28,6 +28,10 @@ extern "C" {
 
 #define TA_VERSION "tangle-accelerator/0.5.0"
 #define TA_HOST "localhost"
+#ifdef ENABLE_MQTT
+#define MQTT_HOST "localhost"
+#define TOPIC_ROOT "NB/root/topics"
```

Comment:
I wonder if `NB` should be the prefix or not.

Reply:
Let me remove it.